specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/CommonDefinitions.json

```diff
@@ -50,6 +50,18 @@
           "description": "Container image tag.",
           "type": "string"
         },
+        "imageType": {
```

Comment:
>imageType [](http://example.com/codeflow?start=9&length=9)

This change looks breaking

Reply:
Is this updating an API version which hasn't shipped yet??

Reply:
> Is this updating an API version which hasn't shipped yet??

yes